docraptor.yaml

```diff
@@ -144,6 +144,9 @@ definitions:
       - document_type
       - document_content
     properties:
+      pipeline:
+        type: string
```

Comment:
Looks like the type parameter change mixed into this PR. No code level changes though so it shouldn't be an issue.